### PR TITLE
CMake fixes

### PR DESCRIPTION
**Pull request by orenh1**
A couple of small fixes to the CMake scripts.

The commit that changes "resolved_item" from a warning to a status message was copied from @jherico: https://github.com/jherico/OculusMinimalExample/blob/master/cmake/templates/FixupBundlePostBuild.cmake.in

About the fix for 'oven' on Linux: CMake failed until I made this change. It was fine on Windows though.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can one of the admins verify this patch?

**Comment by sethalves** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8860/app_12917_PR_50eaf3e166bbe76cac76214788be2c4120cd1e9b.apk

File: cmake/templates/FixupBundlePostBuild.cmake.in [hunk 1]
```diff
@@ -11,6 +11,36 @@
...
+  endif()
+
+  if ("${resolved_item_compare}" STREQUAL "${resolved_embedded_item_compare}")
+    # this is our only change from the original version
+    message(STATUS "status: resolved_item == resolved_embedded_item - not copying...")
+  else()
+    #message(STATUS "copying COMMAND ${CMAKE_COMMAND} -E copy ${resolved_item} ${resolved_embedded_item}")
+    execute_process(COMMAND ${CMAKE_COMMAND} -E copy "${resolved_item}" "${resolved_embedded_item}")
+    if(UNIX AND NOT APPLE)
+        file(RPATH_REMOVE FILE "${resolved_embedded_item}")
```
Comment (birarda) [inline cmake/templates/FixupBundlePostBuild.cmake.in]: two spaces here and a space after the if on line 38

File: cmake/templates/FixupBundlePostBuild.cmake.in [hunk 2]
```diff
@@ -11,6 +11,36 @@
...
+  else()
+    set(resolved_item_compare "${resolved_item}")
+    set(resolved_embedded_item_compare "${resolved_embedded_item}")
+  endif()
+
+  if ("${resolved_item_compare}" STREQUAL "${resolved_embedded_item_compare}")
+    # this is our only change from the original version
+    message(STATUS "status: resolved_item == resolved_embedded_item - not copying...")
+  else()
+    #message(STATUS "copying COMMAND ${CMAKE_COMMAND} -E copy ${resolved_item} ${resolved_embedded_item}")
```
Comment (birarda) [inline cmake/templates/FixupBundlePostBuild.cmake.in]: If this isn't needed other than for debugging you can remove it

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8860/HighFidelity-Beta-PR12917-50eaf3e166bbe76cac76214788be2c4120cd1e9b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8860/HighFidelity-Beta-Interface-PR12917-50eaf3e166bbe76cac76214788be2c4120cd1e9b.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8889/app_12917_PR_cd06067030ad1d7ab791361df35aa41fd456bf11.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8889/HighFidelity-Beta-PR12917-cd06067030ad1d7ab791361df35aa41fd456bf11.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8889/HighFidelity-Beta-Interface-PR12917-cd06067030ad1d7ab791361df35aa41fd456bf11.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8889/HighFidelity-Beta-PR12917-cd06067030ad1d7ab791361df35aa41fd456bf11.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8889/HighFidelity-Beta-Interface-PR12917-cd06067030ad1d7ab791361df35aa41fd456bf11.exe

**Comment by birarda** [comment]
Build test sufficient here, QA not required

**Pull request merged**